ui/components/app/add-network/add-network.js

```diff
@@ -63,29 +69,70 @@ const AddNetwork = ({
           color={COLORS.UI3}
           margin={[4, 0, 3, 0]}
         >
-          {t('customNetworks')}
+          {t('popularCustomNetworks')}
         </Typography>
         {nets.map((item, index) => (
           <Box
             key={index}
             display={DISPLAY.FLEX}
             alignItems={ALIGN_ITEMS.CENTER}
+            justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
             marginBottom={6}
           >
-            <img
-              className="add-network__token-image"
-              src={item?.rpcPrefs?.imageUrl}
-              alt={t('logo', [item.ticker])}
-            />
-            <Typography variant={TYPOGRAPHY.H7} color={COLORS.BLACK}>
-              {item.ticker}
-            </Typography>
-            <img
-              className="add-network__add-icon"
-              src="./images/times.svg"
-              alt={`${t('add')} ${item.ticker}`}
-              onClick={onAddNetworkClick}
-            />
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              <IconBorder size={24}>
+                <IconWithFallback
+                  icon={item.rpcPrefs.imageUrl}
+                  name={item.nickname}
+                  size={24}
+                />
+              </IconBorder>
+              <Typography
+                variant={TYPOGRAPHY.H7}
+                color={COLORS.BLACK}
+                fontWeight={FONT_WEIGHT.BOLD}
+                boxProps={{ marginLeft: 2 }}
+              >
+                {item.nickname}
+              </Typography>
+            </Box>
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              {!regexp.test(item.rpcUrl) && (
+                <Tooltip
+                  className="add-network__warning-tooltip"
+                  position="top"
+                  interactive
+                  html={
+                    <Box margin={3} className="add-network__warning-tooltip">
+                      {t('addNetworkTooltipWarning', [
+                        <a
+                          key="zendesk_page_link"
+                          href="https://metamask.zendesk.com/hc/en-us/articles/4417500466971"
+                          rel="noreferrer"
+                          target="_blank"
+                        >
+                          {t('learnMoreUpperCase')}
+                        </a>,
+                      ])}
+                    </Box>
+                  }
+                  trigger="mouseenter"
+                  theme="light"
+                >
+                  <img
+                    className="add-network__warning-image"
+                    src="images/warning-triangle-grey.svg"
+                    alt=""
```

Comment:
not sure we need an `alt` attribute here

Reply:
Removed

Reply:
An empty `alt` attribute signals to screen readers to not announce the image at all, whereas not having an `alt` attribute at all sometimes triggers the screenreader to read the image URL.